scripts/ci-e2e-lib.sh

```diff
@@ -87,9 +87,9 @@ k8s::resolveVersion() {
   fi
 
   if [[ "$version" =~ ^ci/ ]]; then
-    resolveVersion=$(curl -LsS "http://gcsweb.k8s.io/gcs/kubernetes-release-dev/ci/${version#ci/}.txt")
+    resolveVersion=$(curl -LsS "http://dl.k8s.io/ci/${version#ci/}.txt")
```

Comment:
FYI: this might still be causing you to pickup stale builds depending on how racy/flaky the build and build-deprecated jobs are.  You're getting the version marker from gs://kubernetes-release-dev, but the binaries from gs://k8s-release-dev

Using dl.k8s.io for both means you don't have to think about it